src/lib.rs

```diff
@@ -751,6 +846,34 @@ pub unsafe trait FromZeroes {
     /// Self::new_zeroed()`, it differs in that `zero` does not semantically
     /// drop the current value and replace it with a new one - it simply
     /// modifies the bytes of the existing value.
+    ///
+    /// ```
+    /// # mod derive { pub use zerocopy_derive::FromZeroes; }
+    /// # use zerocopy::FromZeroes;
+    /// # use derive::*;
+    /// #
+    /// # #[derive(FromZeroes)]
+    /// # #[repr(C)]
+    /// # struct PacketHeader {
```

Comment:
So long as we're using a network example, we might as well use a real one.

```suggestion
    /// # struct UdpHeader {
```

Reply:
The name is intentionally obfuscated, as our doc examples aren't meant to be a piecemeal guide for implementing a UDP (de)serializer — they're meant only to provide a gist of how to use the documented function. There are surely going to be complexities in parsing `UdpHeader`s that we might want to elide for these examples. For instance, in this case, I'm not using the byteorder types.

Reply:
Discussed offline; I'm on board with leaving this as-is.